src/inspector/models/abstractinspectormodel.cpp

```diff
@@ -179,15 +179,19 @@ void AbstractInspectorModel::onCurrentNotationChanged()
         onNotationChanged({}, {});
     });
 
-    notation->undoStack()->changesChannel().onReceive(this, [this](const ChangesRange& range) {
-        if (range.changedPropertyIdSet.empty() && range.changedStyleIdSet.empty()) {
+    notation->undoStack()->changesChannel().onReceive(this, [this](const ScoreChanges& changes) {
+        if (changes.isTextEditing) {
```

Comment:
For testing: need to make sure that the Text section of the properties panel still properly updates when using formatting shortcuts or when using the arrow keys to navigate through formatted text.